.travis.yml

```
@@ -8,7 +8,7 @@ matrix:
     - php: 7.1
       env: SYMFONY_VERSION=3.4.*
     - php: 7.1
-      env: SYMFONY_VERSION=4.0.*
+      env: SYMFONY_VERSION=4.*
```

By changing this to `4.*` you actually made this running in the same Symfony environment as the default PHP 7.1 job. I noticed that the PHP CS Fixer is the one conflicting with running in with different Symfony versions at the moment.

This should stay:
```suggestion
      env: SYMFONY_VERSION=4.0.*
```

---

Changing line 18 of this file to the following should resolve the versioning conflict:
```yaml
    - if [ "$SYMFONY_VERSION" != "" ]; then composer remove friendsofphp/php-cs-fixer --dev --no-update; composer require "symfony/symfony:${SYMFONY_VERSION}" --no-update; composer update symfony/*; fi;
```